File: Listas de Exercícios/Listas de Exercícios (Python)/Python/Lista 3 - Casa/10.py
print("Calculadora")

while True:
    print("\n\n============================================================")
    print("info: Escolha a operação com base no símbolo em parênteses!\n")
    print(" S=Sair do Programa")
    print("--------------------")
    print("   Soma       |  (+)")
    print(" Subtração    |  (-)")
    print("Multiplicação |  (*)")
    print("  Divisão     |  (/)")
    print("--------------------")
    OP=str(input("Operação: "))

    if(OP=="S"):
        break

    print("\n--Valores decimais devem ser digitados com (.)!--")
    print("Digite os valores que serão utilizados no cálculo, em ordem!")
    N1=float(input("1º Valor: "))
    N2=float(input("2º Valor: "))

    print("")

    if (OP=="+"):
        print(f"{N1} + {N2} = {N1+N2}")
    elif (OP=="-"):
        print(f"{N1} - {N2} = {N1-N2}")
    elif (OP=="*"):
        print(f"{N1} * {N2} = {N1*N2}")
    elif (OP=="/"):
        print(f"{N1} / {N2} = {N1/N2}")

print("")

File: 1th grade/Listas de Exercícios (Back-end [SPOLOGP])/C/Lista 3 - Casa/10.c
#include <stdio.h>

int main()
{
    char OP;
    float N1, N2;
    printf("Calculadora\n\n");
    
    do {
        printf("\n\n============================================================\n");
        printf("info: Escolha a operação com base no símbolo em parênteses!\n\n");
        printf(" S=Sair do Programa\n");
        printf("--------------------\n");
        printf("   Soma       |  (+)\n");
        printf(" Subtração    |  (-)\n");
        printf("Multiplicação |  (*)\n");
        printf("  Divisão     |  (/)\n");
        printf("--------------------\n");
        printf("Operação: "); scanf("%s", &OP);
        
        if (OP == 'S' || OP == 's'){
            break;
        }
        
        printf("\n--Valores decimais devem ser digitados com (.)!--\n");
        printf("Digite os valores que serão utilizados no cálculo, em ordem!\n");
        printf("1º Valor: "); scanf("%f", &N1);
        printf("2º Valor: "); scanf("%f", &N2);
        printf("\n");
        
        if (OP == '+'){
            printf("%.2f + %.2f = %.2f", N1, N2, N1+N2);
        }else if (OP=='-'){
            printf("%.2f - %.2f = %.2f", N1, N2, N1-N2);
        }else if (OP=='*'){
            printf("%.2f * %.2f = %.2f", N1, N2, N1*N2);
        }else if (OP=='/'){
            printf("%.2f / %.2f = %.2f", N1, N2, N1/N2);
        }
    } while (1);
}